Code/Tools/AssetProcessor/native/FileWatcher/FileWatcher.cpp

```diff
@@ -164,9 +129,11 @@ void FileWatcher::StartWatching()
         return;
     }
 
-    for (FolderRootWatch* root : m_folderWatchRoots)
+    if (PlatformStart())
     {
-        root->Start();
+        m_thread = AZStd::thread([this]{
```

Comment:
While we're here, it'd be great if we could use the `AZStd::thread` constructor that takes a `thread_desc` so we can supply a thread name. In fact, I'd like to advocate that *all* threads in the engine are named adequately (so they are easily identified in profiles/captures)

Reply:
Ha, I looked for a way to name the thread and didn't see it. I'll add it.